docs/overview/key_components/digital_assets.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: i18n_page
+title: pages.digital_assets
+parent: pages.key_features
+grand_parent: pages.understanding_firefly
+nav_order: 2
+---
+
+# Digital Assets
+{: .no_toc }
+
+---
+
+![Hyperledger FireFly Digital Asset Features](../../images/firefly_functionality_overview_digital_assets.png)
+
+## Digital asset features
+
+The modelling, transfer and management of digital assets is the core programming
+foundation of blockchain.
+
+Yet out of the box, raw blockchains designed to efficiently manage these assets
+in large ecosystems, do not come with all the building blocks needed by applications.
+
+### Token API
+
+Token standards have been evolving in the industry through standards
+like ERC-20/ERC-721, and the Web3 signing wallets and that support these.
```

Comment:
`and the Web3 signing wallets and that support these.` unnecessary second `and`.